apps/root/src/pages/transfer/transfer-container/components/wallet-selector/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
```

Comment:
Nice component! It could go directly into `common` since we'll probably use it in other products